kie-aries-blueprint/src/main/java/org/kie/aries/blueprint/KieNamespaceHandler.java

```diff
@@ -43,7 +55,6 @@
     private static final String BLUEPRINT_NS = "http://www.osgi.org/xmlns/blueprint/v1.0.0";
 
     public static final String ELEMENT_RELEASE_ID = "releaseId";
-    public static final String ELEMENT_KBASE_REF = "kbase-ref";
```

Comment:
Just to recap: we are removing the _kbase-ref_ element, because the preferred way of using KieBases/KieSessions from a kJar is by _import_ element. In addition, _ksession-ref_ element is preserved because of possible need to inject _TransactionManager_, _UserTransaction_, etc. into it. Is this correct?